addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -3,7 +3,7 @@
 
 <t t-name="mail.discuss" owl="1">
     <div class="d-flex h-100 flex-grow-1">
-        <Sidebar t-if="props.sidebar"/>
+        <Sidebar t-if="!props.guest"/>
```

Comment:
No need this props. Can rely on `store.currentGuest`

Reply:
Then we need to handle the currentGuest better because from what I see, when `data = ['clear']`, the currentGuest will be created as well, which leading to an `undefined id` but existing `currentGuest`

Reply:
I don't see how you can get `['clear']`, as I cannot reproduce and see no occurrences in the code.
But if we had such payload, then it should not create the `currentGuest` at all.